cli/internal/plugins/bzr/bzr_versioner.go

```diff
@@ -0,0 +1,48 @@
+package bzr
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strings"
+)
+
+// BzrVersioner ...
+type BzrVersioner struct {
+	versionRunner VersionRunner
```

Comment:
Look at how build, test, generate, etc... are doing this. They search through the plugins to find an implementation.